.github/workflows/rolling_build.yml

```diff
@@ -45,11 +46,17 @@ jobs:
     # Trigger the other workflow if the CRM data was updated
     - name: Build and deploy People
       if: steps.determine-deployment-trigger.outputs.should_trigger_deploy == 'TRUE'
-      uses: ./.github/workflows/main-trigger-people-deploy.yml
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: main-trigger-people-deploy.yml
+        repo: SSWConsulting/SSW.People.Profiles
+        ref: ${{ steps.determine-deployment-trigger.outputs.branch_name }}
+        token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
can we pass something so that it's easy to identify that it was triggered by a change in CRM skills?

Reply:
As per our conversation, I've added https://github.com/SSWConsulting/SSW.People/issues/655